card/mod/04_settings/set/self/recaptcha_public_key.rb

```diff
@@ -0,0 +1,9 @@
+event :update_recaptcha_public_key, :after=>:store do
```

Comment:
I'm pretty confused about these new configurations.  Are they Settings or not?  In the migration, it looks like they're Phrase cards, but here we're using `extend Card::Setting`.  A recaptcha public key isn't set-specific, is it?

If not, why do we assign them to groups?